test/syscalls/linux/BUILD

```diff
@@ -3322,7 +3328,10 @@ cc_binary(
 cc_binary(
     name = "sysret_test",
     testonly = 1,
-    srcs = ["sysret.cc"],
+    srcs = select_arch(
```

Comment:
it looks like sysret can be used on arm64. We only need to define macros to set instruction and stack registers. Do I miss something?

Reply:
There is no "sysret" instruction on arm64. I think this test is dedicated to x86. Or we may add a similar test for "eret" instruction on arm64.

Reply:
sysret is called in the kernel space, this test doesn't call this instruction directly. Will test cases for sysret and eret be the same? If the answer is no, could you explain a difference?

Reply:
Just quickly look through the the test, and I think the logic is the same for both platform. But several difference are there:
1. user_regs_struct.rip/rsp -> this can be fixed by conditional compile;
2. PTRACE_SETREGS/GETREGS are not supported on arm64. -> need to change the ptrace call, or may be change to use PTRACE_SETREGSET/GETREGSET.

But to enable the CI system on arm64, I think it's Ok just skip it currently, and add the support later. What's your opinion?

Reply:
I'm ok to skip it now, but could you file a new issue to enable it in a future?

Reply:
Ok!